services/vibration.md

```diff
@@ -10,3 +10,12 @@ Control a vibration motor.
 
 Rotation speed of the motor.
 If only one rotation speed is supported, then `0` shell be off, and any other number on.
+
+## Commands
+
+    command vibrate @ 0x80 {
```

Comment:
maybe we should really do vibration patterns, like you said before...

Reply:
yes!

Reply:
and it should be finite. you don't want to have it vibrate forever becaues of a bug

Reply:
```
{
repeats:
    intensity: u8 /
    duration: u8 8ms
}
```